packages/algob/src/lib/constants.ts

```diff
@@ -3,6 +3,6 @@ import { Algodv2 } from "algosdk";
 export const globalZeroAddress = "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY5HFKQ";
 export const MIN_UINT64 = 0n;
 export const MAX_UINT64 = 0xffffffffffffffffn;
-export const WAIT_ROUNDS = 6; // for transaction confirmation
+export const WAIT_ROUNDS = 10; // for transaction confirmation
```

Comment:
why increasing? I think 6 was long enough. Also in `sendAndWait` documentation note what's the default

Reply:
Because in algob/web we use WAIT_ROUNDS = 10

Reply:
then let's import the variable from `web` here.